.github/workflows/e2eLocal.yml

```diff
@@ -151,7 +137,7 @@ jobs:
         echo "::endgroup::"
 
     - name: Archive cypress failures
-      if: failure()
+      if: always()
```

Comment:
Always save the videos (they are always recorded anyway)
Video for successful runs is helpful with debugging failed runs, let me know if you have any concerns